src/pages/en/core-concepts/sharing-state.md

```diff
@@ -25,14 +25,26 @@ Still, there are a number of alternatives you can explore. These include:
 - [Solid signals](https://www.solidjs.com/docs/latest) outside of a component context
 - [Sending custom browser events](https://developer.mozilla.org/en-US/docs/Web/Events/Creating_and_triggering_events) between components
 
-:::note[Comparisons]
+:::note[FAQ]
+
+<details>
+<summary>**🙋 Can I use Nano Stores in `.astro` files or other server-side components?**</summary>
+
+Nano Stores _can_ be imported, written to, and read from in server-side components, **but we don't recommend it!** This is due to a few restrictions:
+- Writing to a store from a `.astro` file or [non-hydrated component](/en/core-concepts/framework-components/#hydrating-interactive-components) will _not_ affect the value received by [client-side components](/en/reference/directives-reference/#client-directives).
+- You cannot pass a Nano Store as a "prop" to client-side components.
+- You cannot subscribe to store changes from a `.astro` file, since Astro components do not rerender.
```

Comment:
I would change this to "re-render" instead, so it's consistent with the other pages.
```suggestion
- You cannot subscribe to store changes from a `.astro` file, since Astro components do not re-render.
```